### PR TITLE
Remade dates fields in trip entity

### PR DESCRIPTION
**Pull request by AndriiChepeliuk**

### CODE REVIEW

**Review by Grasmus — CHANGES_REQUESTED** [review]
Renew validation according to changes
image

File: Core/Specifications/TripSpecification.cs [hunk 1]
```diff
@@ -43,14 +43,12 @@ public GetValidTripById(int tripId, float totalWeight)
         internal class GetByTimeSpace : Specification<Trip>, ISingleResultSpecification<Trip>
```
Comment (Grasmus) [inline Core/Specifications/TripSpecification.cs]: It no longer works with time space. Rename it

File: Core/Specifications/OfferSpecification.cs [hunk 2]
```diff
@@ -21,13 +21,13 @@ public GetById(int offerId, string userId)
                     .Include(o => o.GoodCategory);
             }
 
-            public GetById(int offerId, int tripId, DateTimeOffset expirationTrip)
```
Comment (Grasmus) [inline Core/Specifications/OfferSpecification.cs:24]: Rename it according to offers are pulled from database

**Review by marynakernychna — COMMENTED** [review]
I didn't find anything except the above

**Review by Grasmus — APPROVED** [review]

**Review by marynakernychna — APPROVED** [review]

**Review by Grasmus — DISMISSED** [review]

**Review by Grasmus — APPROVED** [review]

**Pull request merged**